kolibri_explore_plugin/apps/README.md

```diff
@@ -0,0 +1,8 @@
+# Bundle Apps
+
+All custom presentation app bundles should be added in this directory before
+bundle the wheel file.
+
+You can find the custom channel presentation vue applications in this repository:
+
+https://github.com/endlessm/kolibri-channel-custom-web-app
```

Comment:
I fear this README might be hard to find. I would expect this to be explained in the main README. In case the main README grows too much, we can add a `docs/` with multiple markdown files, linked from the main one (like we did for hack-web).

To keep the empty directory, the convention is to add an empty ` .gitkeep` to git revision. Like the `docker/mnt/.gitkeep` in the kolibri repo.